src/Shared/Telemetry/TelemetryProcessor.ts

```diff
@@ -115,6 +116,9 @@ export default class TelemetryProcessor {
 
   private static getData(data?: any): any {
     return {
+      // TODO: Need to `any` here since the window imports Explorer which can't be in strict mode yet
+      authType: (window as any).authType,
+      subscriptionId: userContext.subscriptionId,
```

Comment:
No need to do anything now, but you will eventually need to introduce an "account context" where subscrptionId belongs. It will be necessary once we support more than one account.